tv_apps/tv_shared/js/scrollable.js

```diff
@@ -287,13 +322,11 @@
                       parseInt(startNode.dataset.idx, 10) : this.nodes.length;
       this.nodes.splice(newIdx, 0, newNode);
       this.listElem.appendChild(newNode);
-      this._setNodesPosition();
+      this._setNodePosition(newIdx);
```

Comment:
So, by the implement now, `insertNodeBefore` wouldn't work correctly without calling `slide` since nobody sets position of remaining Nodes.
Please consider if we can call `slide()` inside `insertNodeBefore()`.
If you need to add/remove classes from `home.js` side, we may consider sending events from scrollable when needed (e.g. transitionEnd).